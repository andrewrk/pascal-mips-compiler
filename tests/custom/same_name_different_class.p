PrOgRaM MAiNE;
clAss maIne begin
    function MainE;
        var b : boolean;
        var z : otherdude;
    begin
        ;;;;;;;;;;;;
        z.foo(b);
        foo(b);
    end
end
class blahblah begin
    function foo(z:integer); begin
        print 'hi'
    end
end
class otherDude begin
    function foo(z:boolean); begin
        print 'blah'
    end
end
.
