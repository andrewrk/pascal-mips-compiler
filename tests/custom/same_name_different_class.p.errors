ERROR: line 9: class "maIne" has no method "foo"
