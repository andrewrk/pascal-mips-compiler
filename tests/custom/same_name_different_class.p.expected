ERROR: line 9: function "foo" not declared
